src/definition/accessors/ILivechatRead.ts

```diff
@@ -0,0 +1,7 @@
+import { ILivechatRoom } from '../livechat/ILivechatRoom';
+import { IVisitor } from '../livechat/IVisitor';
+
+export interface ILivechatRead {
+    getLivechatRoom(visitor: IVisitor, departmentId?: string): Promise<Array<ILivechatRoom>>;
```

Comment:
Should these both be plural? `Rooms` and `Visitors` since they both return an array?